CONTRIBUTING.md

```diff
@@ -10,48 +10,53 @@ This document is to act as a guideline for how you should submit Pull Requests.
 
 # <a name="introduction"></a>Introduction
 
-This document aims to clarify the coding standard required for a Pull Request to be accepted. 
+This document aims to clarify the coding standard required for a Pull Request to be accepted.
 
 While not all code submitted at this time conforms to this standard, it is the intention of the project to correct it in time.
 
 
 # <a name="coding-style-and-formatting"></a>Coding Style and Formatting
 
+- [Auto formatting](#auto-formatting)
 - [Naming](#naming)
 - [Structs](#structs)
 - [Functions](#functions)
 - [Conditionals](#conditionals)
+- [Pull Requests](#prs)
+
+## <a name="auto-formatting"></a>Auto Formatting
+C code that's contributed to this codebase should have `clang-format` ran on it.
```

Comment:
Some mention of [`git clang-format`](https://ortogonal.github.io/cpp/git-clang-format/) would be useful here, too. And maybe a link to a guide on how to set it up in VS Code, since that's what most people use.

Reply:
I honestly have no idea how to use VSCode, I'm probably not the best for that type of documentation.

Didn't know about `git clang-format`. learned and added.